packages/babel-helpers/src/index.js

```diff
@@ -178,13 +185,16 @@ function permuteHelperAST(file, metadata, id, getLocalBindings, getDependency) {
       const impsBindingRefs = importBindingsReferences.map(p => path.get(p));
 
       const decl = exp.get("declaration");
+
       if (id.type === "Identifier") {
         if (decl.isFunctionDeclaration()) {
           exp.replaceWith(decl);
-        } else {
+        } else if (!isKeywordHelper) {
           exp.replaceWith(
             t.variableDeclaration("var", [t.variableDeclarator(id, decl.node)]),
           );
+        } else {
+          exp.remove();
```

Comment:
Could you help clarify, what specifically is this changing? I actually didn't realize the `keywordHelpers` whitelist existed, and the fact that it does seems like an issue we should fix if we can.

Reply:
> Could you help clarify, what specifically is this changing? 

Its a leftover from me trying to change self-redeclaring helpers. Gonna remove this line in a sec.

> I actually didn't realize the keywordHelpers whitelist existed, and the fact that it does seems like an issue we should fix if we can.

Any proposal how this should get fixed? @hzoo proposed that usage (i.e. `this.addHelper('extends')`) should stay the same